src/org/daisy/dotify/api/formatter/TextProperties.java

```diff
@@ -96,11 +109,21 @@ public boolean isHyphenating() {
 		return hyphenate;
 	}
 
+    /**
+     * Returns true if the mark capital letters property is true, false otherwise
+     * @return returns true if the mark capital letters property is true
+     */
+    public boolean shouldMarkCapitalLetters() {
```

Comment:
Hi @PaulRambags

When it comes to the naming of this function, I use the most reasonable verb. If you want another example of this use, you have it with the hyphenate function: https://github.com/mtmse/dotify.library/blob/0e7b130d72a2355b995a7a0c97abe821c065dce0/src/org/daisy/dotify/translator/DefaultBrailleFilter.java#L128

In the specific case, the boolean doesn't tell you if the current object is a mark capital letter. "is" as a keyword is really great for telling about a state of an object. But in this case, it tells us if we should run a specific piece of code.

The style guide of Oracle only tells you to use a relevant verb for your function names:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

This can be a part that we could debate what verbs are allowed in this codebase, and then I will change it. But in that change, I will go through and change all other usages of the same keywords, so we get a cohesive use.

Best regards
Daniel